README.md

```diff
@@ -202,13 +202,17 @@ repeating the mistakes of the past.
 - Write blog posts or speak at meetups and conferences.
 
 #### Collaboration over competition
-- We have a very collaborative culture. There are a number of reasons why this is preferable:
-    - [Competitive cultures tend to not bode well for creating diverse teams.](https://medium.com/@racheltho/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996#c63a).
+- We have a very collaborative culture. [Competitive cultures tend to not bode well for creating diverse teams](https://medium.com/@racheltho/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996#c63a), which we value highly.
 
 #### Favour experimentation over argument
 
 #### Be excellent to each other!
 
+#### Inclusion over exclusion
+- We value diversity - and a diverse culture is one that does not exclude people. Bullying,
```

Comment:
:+1: , and that's a thought-provoking article in several ways.

What I saw as the key points in the article:
* When Aurynn realized her comments were having a negative effect, she changed, and stopped.
* She calls out _contempt_ and _ridicule_: those definitely have no part in what we do and how we do it.

However, contrary to Aurynn's article, where she says "even if we didn’t intend to (exclude), _it does not matter_"... well, yes it *does* matter. Intention is important. It's not everything, and it's not necessarily an excuse, but it shouldn't be ignored.

* *Intentionally* expressing contempt, exclusion or ridicule is bullying, and is unacceptable.
* Offending or upsetting other people and treating it as *their* problem (_you should just lighten up_, _don't be so sensitive_, _I was only having a laugh_) is being a Class A jerk. Also unacceptable.
* *Accidentally* upsetting other people is part of being human. We learn that we've done it, and we try to change. We learn that other people do it to us, and we learn to communicate, and try to change.

It's easy to fall into the pattern of writing more and more rules to try and capture any action that might generate negative feelings in other people: as engineers we're probably prone to try and do that, believing that it's possible to come up with some way to control it. I don't believe that is possible, but I *do* think its reasonable to always *consider* other people's feelings, and if you realise that you've upset someone, share the responsibility for that, try and understand (and value) their point of view, and maybe learn something.

Too much complexity for a PR comment on the developer values? Yes, absolutely. But my key point is that *we should be wary of trying to make simple rules to govern complex human interactions*.

So: rather than focus only on exclusion, maybe we should also emphasize inclusion:
* https://hbr.org/2014/08/whos-being-left-out-on-your-team/
* http://studentguide.interactivityfoundationwiki.wikispaces.net/Section+3.7+-+Making+New+Members+Feel+Part+of+the+Team
* http://www.fastcompany.com/3008216/tracking/minding-gap-how-your-company-can-woo-female-coders